grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -911,14 +911,19 @@ class QueryService implements Status {
                 }
 
                 // Exclude abandoned by default unless statuses are specifically requested
-                if (options.ignoreAbandoned && !options.issueStatusNames) {
+                if (options.ignoreAbandoned && !options.issueStatusNames && !options.freeText) {
```

Comment:
Is this necessary? It also renders the comment above it incorrect now, so please update that to reflect the reasoning here.